esp-hal-common/ld/sections/stack.x

```diff
@@ -0,0 +1,36 @@
+#IF flip-link
```

Comment:
Maybe this should just be two files, stack.x and stack-flip-link.x and we do the if statement in the build script?

Reply:
I thought that the pre-processing might give us some nice options in the future to avoid doing "hidden" things in `build.rs`- like I would love to replace this: https://github.com/esp-rs/esp-hal/blob/1130ae4f0d4daf44d6cb172056b8d6162e4fac5f/esp32-hal/build.rs#L21-L36

Reply:
Ah true, I don't feel strongly either way so I'm happy to keep this.